# Duration of the recording in seconds
std_msgs/Bool human_demo
std_msgs/UInt8 nb_points
---
apex_playground/Demonstration demo
